pyproject.toml

```
@@ -38,6 +30,16 @@ cu12 = [
     "nvidia-cuda-nvrtc-cu12",
     "nvidia-nvjitlink-cu12"
 ]
+cu13 = [
+    "cuda-bindings==13.*",
+    "cuda-core==0.3.*",
```

Might need a minimum patch version. Something like this, but check my values for the first patch release to support CUDA 13:
```suggestion
    "cuda-core>=0.3.2,<0.4.0dev0",
```